applications/registration-react/Dockerfile

```diff
@@ -1,18 +1,14 @@
 FROM node:12-alpine
 
 #the only reason why we need git here, is that we have designsystem in another github repo
-RUN apk add --no-cache git
+RUN apk add --no-cache git gettext
 # to run localhost docker
 
-RUN npm init -y
-RUN npm config set unsafe-perm true
-RUN npm install ejs express method-override body-parser compression --loglevel error
-RUN npm install pm2 -g --loglevel error
+WORKDIR /usr/src/app
 
 # Create app directory
-RUN mkdir -p /usr/src/app && chmod 777 /usr/src/app
-RUN mkdir /.pm2 && chmod 777 /.pm2
-WORKDIR /usr/src/app
+#RUN mkdir -p /usr/src/app && chmod 777 /usr/src/app
+#RUN mkdir /.pm2 && chmod 777 /.pm2
```

Comment:
Remove this?

Reply:
jeg prøvde, hvis det funker, og det ser ut sånn.. takk